### PR TITLE
Add svg icon with context-fill color

### PR DESCRIPTION
**Pull request by umpirsky**
Extension icon is black, it's color is different then other icons in the toolbar, I used svg with `fill="context-fill #4c4c4c" fill-opacity="context-fill-opacity"` and `fill="context-fill #BEBEBE" fill-opacity="context-fill-opacity"`to make it same color as others. SVG also scales better.

### Beofre:
screenshot from 2018-02-16 21-35-32

### After:
screenshot from 2018-02-16 21-33-37

### Beofre:
screenshot from 2018-02-19 19-15-56

### After:
screenshot from 2018-02-19 19-14-58

### CODE REVIEW

**Comment by umpirsky** [comment]
@AngelFQC Ping. :)

File: manifest.json [hunk 1]
```diff
@@ -8,14 +8,18 @@
...
         "scripts": ["background.js"]
     },
     "browser_action": {
-        "default_icon": "icons/48/github.png"
+        "default_icon": "icons/github.svg",
+        "theme_icons": [{
+            "light": "icons/github_light.svg",
+            "dark": "icons/github.svg",
+            "size": 64
+        }]
```
Comment (AngelFQC) [inline manifest.json:22]: :+1:

**Comment by umpirsky** [comment]
@AngelFQC Can you merge and release new version? :)

**Comment by AngelFQC** [comment]
I will try to do it tonight

**Comment by AngelFQC** [comment]
I changed the colors according to https://design.firefox.com/photon/visuals/color.html#icons-and-other-elements

**Comment by umpirsky** [comment]
Interesting!

I saw this on https://github.com/mozilla/multi-account-containers/blob/master/src/img/container-site.svg, which is official Firefox addon which looks nice with all themes. Strange they didn't follow photon guidelines.

There is also https://www.npmjs.com/package/photon-colors, not sure how to use it, but I see snooze tabs is using it https://github.com/bwinton/SnoozeTabs/blob/master/package.json#L37.

**Comment by umpirsky** [comment]
@AngelFQC I updated to 1.3.0, still getting black icon?

It looks good with light and dark, but not with default theme:

screenshot from 2018-02-21 14-36-24
screenshot from 2018-02-21 14-36-14
screenshot from 2018-02-21 14-35-56

**Comment by AngelFQC** [comment]
Are you using Fx DevEdition? In Nightly for Ubuntu looks good

**Comment by umpirsky** [comment]
Firefox Quantum 58.0.2 (64-bit) with [Arc Dark theme](https://github.com/horst3180/arc-firefox-theme).

**Comment by umpirsky** [comment]
Reported https://github.com/AngelFQC/firefox-github-notifier/issues/8.

**Pull request merged**